deployment/terraform/modules/rds/main.tf

```diff
@@ -45,3 +45,20 @@ resource "aws_db_instance" "default" {
     ignore_changes = ["final_snapshot_identifier"]
   }
 }
+
+resource "aws_cloudwatch_metric_alarm" "default_rds_cpu_utilization" {
+  alarm_name          = "${var.resource_name_prefix}-default-rds-cpu-utilization"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = "5"
+  metric_name         = "CPUUtilization"
+  namespace           = "AWS/RDS"
+  period              = "300"
+  statistic           = "Average"
+  threshold           = "25"
```

Comment:
Given the data (https://github.com/refinery-platform/refinery-platform/issues/2965#issuecomment-438707280) I'd say a 10% threshold would be more appropriate.

Reply:
Ok

Reply:
does this mean for both ec2 and rds?